### DISCUSSION TITLE
How to get the cluster queue length from the Jolokai console

### DISCUSSION THREAD

**Discussion by lavifb** [arkmq-org/arkmq-org-broker-operator#889] [Q&A]
We have an autoscaling setup using [KEDA](https://keda.sh/docs/2.13/scalers/artemis/) that uses the web console to get the relevant queue length to then scale our consumers for that queue. Unfortunately, the console seems to only show the queue length on that node of the cluster, not the entire aretmis cluster. Is there a way to setup this up so that we get the whole combined queue length on the entire cluster?

**Comment by brusdev** [comment]
There is no way to get the whole combined queue length on the entire cluster but it seems an interesting feature to add, could create a [new enhancement issue](https://github.com/artemiscloud/activemq-artemis-operator/issues/new?labels=enhancement)?

**Comment by lavifb** [comment]
I can do that

**Reply by lavifb** [reply]
https://github.com/artemiscloud/activemq-artemis-operator/issues/892

**Comment by brusdev** [comment]
As already discussed in #892: agents that scrap and aggregate metrics already exist like Prometheus and it is already supported, see https://github.com/artemiscloud/activemq-artemis-operator/blob/main/docs/help/operator.md#monitor-broker-metrics-by-using-prometheus

KEDA already supports Prometheus, see https://keda.sh/docs/1.4/scalers/prometheus/
